app/controllers/accounts/registrations_controller.rb

```diff
@@ -4,10 +4,6 @@ class Accounts::RegistrationsController < Devise::RegistrationsController
   # We delegate the create method to the super class
   # rubocop:disable Rails/LexicallyScopedActionFilter
   before_action :check_captcha, only: [:create]
-  # rubocop:enable Rails/LexicallyScopedActionFilter
```

Comment:
This autocorrect ironically moves a comment that it's not supposed to move, because of a missing blank line